tapestry.js

```diff
@@ -2982,4 +2977,23 @@ $(document).ready(function(){
         var y = event.clientY + $(window).scrollTop();
         recordAnalyticsEvent('user', 'click', 'tapestry', null, {'x': x, 'y': y});
     }, true);
-});
\ No newline at end of file
+});
+
+function getDomNode(idOrClass){
+    const isClass = idOrClass[0] === ".";
+    idOrClass = idOrClass.substring(1);
+    return isClass ? document.getElementByClassName(idOrClass) : document.getElementById(idOrClass);
```

Comment:
Should it be `getElementsByClassName` instead of `getElementByClassName`?

Reply:
In this way, the function may return an array of elements if the argument is a class name.

Reply:
you are right, it has to be `getElementsByClassName`